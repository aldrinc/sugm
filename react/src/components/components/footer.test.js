import React from 'react';
import Footer from './footer';
import renderer from 'react-test-renderer';

it('footer wrenders correctly', () => {
    const tree = renderer.create(
      <Footer></Footer>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

