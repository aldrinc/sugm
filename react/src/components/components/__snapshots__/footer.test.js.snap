// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer wrenders correctly 1`] = `
<div
  className="footer"
>
  <div
    className="footer_mobile"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-sm-12"
        >
          <div
            className="footer_logo"
          >
            <img
              alt=""
              src="footer_logo.jpg"
            />
          </div>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="col-sm-12"
        >
          <div
            className="footer_menu"
          >
            <ul>
              <li>
                <a
                  href="#"
                >
                  Help
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  About us
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Returns
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                   Wholesale
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Privacy Policy
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Give Us Feedback
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Terms
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Email Us
                </a>
              </li>
            </ul>
          </div>
          <div
            className="emailsubscribe"
          >
            <h2>
              Be First
            </h2>
            <p>
              Want exclusive offers and first and access to products? Sign up for email alerts.
            </p>
            <div
              className="emailsubscrivebox"
            >
              <input
                placeholder="Your email address"
                type="text"
              />
              <input
                type="submit"
                value="+"
              />
            </div>
            <div
              className="kjkj"
            >
              <p>
                By entering your email,you agree to our
              </p>
              <p>
                <a
                  href="#"
                >
                  Terms of Service
                </a>
                +
                <a
                  href="#"
                >
                  Privacy Policy
                </a>
                ,including receipt of emails and promotions. You can unsubscribe at any time. 
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="deskfooter"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-md-2"
        >
          <div
            className="footer-title"
          >
            <h5>
              Customer Service
            </h5>
            <ul>
              <li>
                <a
                  href="#"
                >
                  Contact Us
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Order Status
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Shipping
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Returns & Exchanges
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Give Us Feedback
                </a>
              </li>
            </ul>
          </div>
        </div>
        <div
          className="col-md-2"
        >
          <div
            className="footer-title"
          >
            <h5>
              About Us
            </h5>
            <ul>
              <li>
                <a
                  href="#"
                >
                  Shutupandgiftme Cares
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Site Map
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Get Email Updates
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Blogs + More
                </a>
              </li>
            </ul>
          </div>
        </div>
        <div
          className="col-md-2"
        >
          <div
            className="footer-title"
          >
            <h5>
              Services
            </h5>
            <ul>
              <li>
                <a
                  href="#"
                >
                  Bulk Ordering
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Gift Cards
                </a>
              </li>
            </ul>
          </div>
        </div>
        <div
          className="col-md-2"
        >
          <div
            className="footer-title"
          >
            <h5>
              Shutupandgiftme, Inc.
            </h5>
            <ul>
              <li>
                <a
                  href="#"
                >
                  Investor Relations
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Press Releases
                </a>
              </li>
            </ul>
          </div>
        </div>
        <div
          className="col-md-4"
        >
          <div
            className="footer-title"
          >
            <h6>
              Connect with us!
            </h6>
            <ul
              className="social-icon-ft"
            >
              <li>
                <a
                  href="#"
                >
                  <i
                    className="fab fa-instagram"
                  />
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  <i
                    className="fab fa-pinterest-p"
                  />
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  <i
                    className="fab fa-twitter"
                  />
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  <i
                    className="fab fa-facebook-f"
                  />
                </a>
              </li>
            </ul>
          </div>
        </div>
        <div
          className="col-md-6"
        >
          <div
            className="footer-btm"
          >
            <ul>
              <li>
                <a
                  href="#"
                >
                  Your Privacy Rights
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Term & Conditions
                </a>
              </li>
            </ul>
          </div>
        </div>
        <div
          className="col-md-6"
        >
          <div
            className="footer-btm"
          >
            <p
              className="copy"
            >
              Â© 2018 Shutupandgiftme, Inc.
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
